bm-persona/Fitness/FitnessViewController.swift

```diff
@@ -212,19 +271,40 @@ extension FitnessViewController {
         card.leftAnchor.constraint(equalTo: view.layoutMarginsGuide.leftAnchor).isActive = true
         card.rightAnchor.constraint(equalTo: view.layoutMarginsGuide.rightAnchor).isActive = true
         card.heightAnchor.constraint(equalTo: view.layoutMarginsGuide.heightAnchor).isActive = true
+        gymCard = card
         
-        let table = UITableView()
-        table.delegate = self
-        table.dataSource = self
-        scrollView.addSubview(table)
-        table.translatesAutoresizingMaskIntoConstraints = false
-        table.topAnchor.constraint(equalTo: card.layoutMarginsGuide.topAnchor).isActive = true
-        table.leftAnchor.constraint(equalTo: card.layoutMarginsGuide.leftAnchor).isActive = true
-        table.rightAnchor.constraint(equalTo: card.layoutMarginsGuide.rightAnchor).isActive = true
-        table.bottomAnchor.constraint(equalTo: card.layoutMarginsGuide.bottomAnchor).isActive = true
+        let fitnessLabel = UILabel()
+        fitnessLabel.text = "Fitness Centers"
+        fitnessLabel.font = kHeaderFont
+        fitnessLabel.adjustsFontSizeToFitWidth = true
+        fitnessLabel.textColor = .black
+        fitnessLabel.textColor = Color.blackText
+        card.addSubview(fitnessLabel)
         
-        gymCard = card
-        gymsTable = table
+        setupFilterTableView()
+        card.addSubview(filterTableView)
+        
+        fitnessLabel.translatesAutoresizingMaskIntoConstraints = false
+        fitnessLabel.heightAnchor.constraint(equalToConstant: 30).isActive = true
+        fitnessLabel.leftAnchor.constraint(equalTo: card.layoutMarginsGuide.leftAnchor, constant: 8).isActive = true
+        fitnessLabel.rightAnchor.constraint(equalTo: card.layoutMarginsGuide.rightAnchor).isActive = true
+        fitnessLabel.topAnchor.constraint(equalTo: card.layoutMarginsGuide.topAnchor).isActive = true
+        
+        filterTableView.translatesAutoresizingMaskIntoConstraints = false
+        filterTableView.topAnchor.constraint(equalTo: fitnessLabel.layoutMarginsGuide.bottomAnchor, constant: 14).isActive = true
+        filterTableView.leftAnchor.constraint(equalTo: card.layoutMarginsGuide.leftAnchor).isActive = true
+        filterTableView.rightAnchor.constraint(equalTo: card.layoutMarginsGuide.rightAnchor).isActive = true
+        filterTableView.bottomAnchor.constraint(equalTo: card.layoutMarginsGuide.bottomAnchor, constant: -50).isActive = true
```

Comment:
Remove the constant here as well, but only because the drawer will have its size changed / up position moved to fix the cutting-off issue you see here.